qa/tasks/qemu.py

```diff
@@ -87,8 +88,24 @@ def generate_iso(ctx, config):
     """Execute system commands to generate iso"""
     log.info('generating iso...')
     testdir = teuthology.get_testdir(ctx)
+
+    refspec = ctx.config.get('branch')
```

Comment:
Hmm --- perhaps this is supposed to be just ```config``` instead of ```ctx.config```?  The test you ran should have used the sha1 but instead it used ```HEAD``` revision